numba_dpex/core/runtime/context.py

```diff
@@ -475,16 +477,18 @@ def acquire_meminfo_and_schedule_release(
     def build_or_get_kernel(self, builder: llvmir.IRBuilder, args):
         """Inserts LLVM IR to call build_or_get_kernel.
 
-        DPCTLSyclKernelRef
-        DPEXRT_build_or_get_kernel(
-            const DPCTLSyclContextRef ctx,
-            const DPCTLSyclDeviceRef dev,
-            size_t il_hash,
-            const char *il,
-            size_t il_length,
-            const char *compile_opts,
-            const char *kernel_name,
-        );
+        .. code-block::
```

Comment:
```suggestion
        .. code-block:: c
```

Default code-block highlighting is for Python